data/Akash-vadakkeveetil.json

```
@@ -0,0 +1,32 @@
+{
+    "name": "Akash Vadakkeveetil",
+    "bio": "Open source contributions and DevOps enthusiast",
+    "tags": [
+        "Open Source",
+        "Javascript",
+        "Java",
+        "DevOps"
+    ],
+    "socials": [
+        {
+            "icon": "FaTwitter",
+            "url": "https://twitter.com/Akash_V_V"
+        },
+        {
+            "icon": "FaGithub",
+            "url": "https://github.com/Akash-vadakkeveetil"
+        },
+        {
+            "icon": "FaLinkedin",
+            "url": "https://www.linkedin.com/in/akash-vadakkeveetil"
+        }
+    ],
+    "links": [
+        {
+            "group": "Blogs",
+            "name": "Learn more about Tech",
+            "url": "https://hashnode.com/@Akash-Vadakkeveetil",
+            "icon": "FaHashode"
```

FaHashode icon doesn't exists.

---

@Akash-vadakkeveetil It should be `SiHashnode`.

```suggestion
            "icon": "SiHashode"
```